crates/config/src/toolchain/python_config.rs

```diff
@@ -0,0 +1,33 @@
+// use super::bin_config::BinEntry;
+use schematic::Config;
+use serde::Serialize;
+use version_spec::UnresolvedVersionSpec;
+use warpgate_api::PluginLocator;
+
+#[derive(Clone, Config, Debug, PartialEq, Serialize)]
+pub struct PipConfig {
+    /// List of arguments to append to `pip install` commands.
+    pub install_args: Option<Vec<String>>,
+}
+
+#[derive(Clone, Config, Debug, PartialEq)]
+pub struct PythonConfig {
+    /// Location of the WASM plugin to use for Python support.
+    pub plugin: Option<PluginLocator>,
+
+    /// Options for pip, when used as a package manager.
+    #[setting(nested)]
+    pub pip: Option<PipConfig>,
+
+    #[setting(default = ".venv", skip)]
```

Comment:
Can you add a description then run `just schemas`.

Reply:
Will do